src/core/public/chrome/ui/header/nav_link.tsx

```diff
@@ -110,12 +109,11 @@ export interface RecentNavLink {
  * @param navLinks
  * @param basePath
  */
-export function createRecentNavLink(
+export function createRecentChromeNavLink(
```

Comment:
I'm still looking at the this, please don't merge for now, having a few doubts need to figure out